subtests/docker_cli/systemd_in_container/test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#
+# Test systemd in containers
+#
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 IMAGE-NAME"
+    exit 1
+fi
+image="$1"
+
+# Exit status. Default 0 (success), but set to 1 on any subtest failure
+rc=0
+
+# Main code. Run a command whose output is expected to be a single line,
+# and the first field is a security context (user:role:type:level); verify
+# that command generates exactly one line of output; extract the <type>
+# and <level> fields; and check against a list of one or more expected
+# values. Assume that there's no overlap between valid types & ranges.
+check_output() {
+    local name="$1"; shift
+    local command="$1"; shift
+
+    $command < /dev/ptmx > /tmp/systemd$$.log 2>&1 &
+    sleep 30
```

Comment:
Many other tests do something very similar by monitoring the container's logs for a keyword.  In fact, it's used so frequently, ``dockertest.output.wait_for_output()`` was written a loooooong time ago.   There's also ``dockertest.AsyncDockerCmd.wait_for_ready()`` for handling this.

Actually, that makes me think.  Maybe this would be easier if you split up the script by phase,  e.g. execute one from e.g. ``initialize()``, another from ``run_once()``, and maybe another from ``cleanup()``.  That would allow for more modular completion-testing of the previous phase, either from script(s) or w/in python (either way is fine).

BTW, we recognize python isn't everyone's cup-o-tea, doing this in bash or any other language available on an Atomic host is totally cool.  Happy to help add necessary hooks to the python module if needed as well.